js_tests/admin/actions.test.js

```diff
@@ -0,0 +1,21 @@
+module('admin.actions', {
+    beforeEach: function() {
+        // Number of results shown on page
+        window._actions_icnt = '100';
+
+        var $ = django.jQuery;
+        $('#qunit-fixture').append($('#result-table').text());
+
+        // DOM needs to be loaded before using actions
+        $(document).ready(function($) {
+            $('tr input.action-select').actions();
```

Comment:
I've just realised that there is no guarantee that this will indeed finish before test is executed. Maybe we need something like `assert.async()` here?